scripts/loki_transform.py

```diff
@@ -69,8 +75,8 @@ def cli(debug):
 @cli.command()
 @click.option('--mode', '-m', default='idem',
               type=click.Choice(
-                  ['idem', "c", 'idem-stack', 'sca', 'claw', 'scc', 'scc-hoist', 'scc-stack',
-                   'cuf-parametrise', 'cuf-hoist', 'cuf-dynamic', 'scc-raw-stack']
+                  ['idem', "c", 'idem-lower', 'idem-lower-loop', 'idem-stack', 'sca', 'claw', 'scc', 'scc-hoist', 'scc-stack',
+                   'cuf-parametrise', 'cuf-hoist', 'cuda-parametrise', 'cuda-hoist', 'scc-raw-stack']
```

Comment:
Since specifying the pipeline in the script is considered deprecated, should we really add new options here or would it make more sense to switch to the config file approach for these? (Cc @mlange05 )